clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapServerStore.java

```diff
@@ -222,4 +223,102 @@ public void close() {
     }
   }
 
+  // stats
+
+  public long allocatedMemory() {
```

Comment:
hmm... so weird.  I guess i missed it.  i thought I had changed it.  Anyways i just ran chmod 644 on it.